crates/common/src/zk_compile.rs

```diff
@@ -913,19 +933,38 @@ impl ZkSolc {
     /// The versioned sources can then be used for further processing or analysis.
     fn get_versioned_sources(&mut self) -> Result<BTreeMap<Solc, SolidityVersionSources>> {
         // Step 1: Retrieve Project Sources
-        let sources = self.project.paths.read_input_files()?;
+        let mut sources = match self.project.sources() {
+            Ok(sources) => sources,
+            Err(e) => return Err(eyre::eyre!("Could not get project sources: {}", e)),
+        };
 
+        let mut script_path = PathBuf::from(self.project.sources_path());
+
+        // Pop the last component ("/src") and then push the new component ("/scripts")
+        if script_path.ends_with("src") {
+            script_path.pop();
+            script_path.push("script");
+        }
+        let sources_two = Source::read_all_from(script_path)?;
+        sources.extend(sources_two);
         // Step 2: Resolve Graph of Sources and Versions
-        let graph = Graph::resolve_sources(&self.project.paths, sources)
-            .wrap_err("Could not resolve sources")?;
+        let graph = match Graph::resolve_sources(&self.project.paths, sources) {
+            Ok(graph) => graph,
+            Err(e) => return Err(eyre::eyre!("Could not resolve sources: {}", e)),
+        };
 
         // Step 3: Extract Versions and Edges
-        let (versions, _edges) = graph
-            .into_sources_by_version(self.project.offline)
-            .wrap_err("Could not match solc versions to files")?;
+        let (versions, _edges) = match graph.into_sources_by_version(self.project.offline) {
+            Ok(sources) => sources,
+            Err(e) => return Err(eyre::eyre!("Could not match solc versions to files: {}", e)),
+        };
 
         // Step 4: Retrieve Solc Version
-        versions.get(&self.project).wrap_err("Could not get solc")
+        match versions.get(&self.project) {
+            Ok(solc) => Ok(solc),
+            Err(e) => Err(eyre::eyre!("Could not get solc: {}", e)),
+        }
+        // versions.get(&self.project).wrap_err("Could not get solc")
```

Comment:
This should be restored to reflect `main` as script dir is included during compiling since: https://github.com/matter-labs/foundry-zksync/pull/210